auto_submit/lib/requests/github_webhook.dart

```diff
@@ -17,7 +19,24 @@ class GithubWebhook {
   Future<Response> post(Request request) async {
     final Map<String, String> reqHeader = request.headers;
     logger.info('Header: $reqHeader');
+
+    // Listen to the pull request with 'autosubmit' label.
+    bool hasAutosubmit = false;
     final String rawBody = await request.readAsString();
+    final body = json.decode(rawBody) as Map<String, dynamic>;
+
+    if (!body.containsKey('pull_request')) {
+      return Response.notFound(
```

Comment:
We want to avoid returning errors on webhook.

In this case, we can return an empty body since this is a no-op for the app